src/Resque/Component/Job/Model/JobInterface.php

```diff
@@ -67,6 +67,8 @@ public function getArguments();
      */
     public function encode();
 
+    public function getOriginQueue();
```

Comment:
Do you know/remember what places you needed this on specifically? Do you feel like I should just have this on JobInterface instead of having OriginQueueAwareInterface?